website/docs/fundamentals/managed-node-groups/spot/index.md

```diff
@@ -3,21 +3,151 @@ title: Spot instances
+eks-workshop    managed-system-20230605211832120700000024               ACTIVE          2023-06-05T21:18:34Z    1               2               1             m5.large                                                 AL2_x86_64      eks-managed-system-20230605211832120700000024-26c446b6-8271-ac8a-4b54-569cf51913f9    managed
+
+```
+
+:::info
+The aws `eks wait nodegroup-active` command can be used to wait until a specific EKS node group is active and ready for use. This command is part of the AWS CLI and can be used to ensure that the specified node group has been successfully created and all the associated instances are running and ready.
+
+```bash
+$ aws eks wait nodegroup-active \
+--cluster-name $EKS_CLUSTER_NAME \
+--nodegroup-name $EKS_DEFAULT_MNG_NAME_SPOT
+```
+:::
+
+Once the Managed node group `managed-spot` status shows as “<b>Active</b>”, Run the below command. 
+The output shows that two additional nodes got provisioned under the node group `managed-spot` with capacity type as `SPOT`.
+
+```bash
+$ kubectl get nodes -L eks.amazonaws.com/capacityType,eks.amazonaws.com/nodegroup
+
+NAME                                         STATUS   ROLES    AGE    VERSION                CAPACITYTYPE   NODEGROUP
+ip-10-42-10-232.us-west-2.compute.internal   Ready    <none>   113m   v1.23.15-eks-49d8fe8   ON_DEMAND      managed-system-20230605211737831800000026
+ip-10-42-10-96.us-west-2.compute.internal    Ready    <none>   113m   v1.23.15-eks-49d8fe8   ON_DEMAND      managed-ondemand-20230605211738568600000028
+ip-10-42-11-17.us-west-2.compute.internal    Ready    <none>   78s    v1.23.17-eks-0a21954   SPOT           managed-spot
+ip-10-42-12-234.us-west-2.compute.internal   Ready    <none>   77s    v1.23.17-eks-0a21954   SPOT           managed-spot
+ip-10-42-12-45.us-west-2.compute.internal    Ready    <none>   113m   v1.23.15-eks-49d8fe8   ON_DEMAND      managed-ondemand-20230605211738568600000028
+
+```
+From the above output you can see that now we have two managed nodegroups available. Next, let’s deploy the “Sample Retail Store” app and leverage [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) as node selection constraint to deploy app on top of the spot instances instead of On-Demand. Using the nodeSelector field, you can specify key-value pairs to filter and select which nodes in the cluster should run particular pods. It is used to define constraints for pod placement based on node labels. 
+
+Now because our existing manifest `deployment.yaml` located under `/workspace/manifests/catalog`  does not have `nodeSelector` as an attribute, we will use [kustomize](https://www.eksworkshop.com/docs/introduction/kustomize/) to modify the existing resource configuration without directly modifying the original manifests. 
```

Comment:
A bit wordy. Can simply say "Run the following command to update the existing deployment.yaml to add `nodeSelector`.